public/app/plugins/datasource/prometheus/querybuilder/shared/QueryOptionGroup.tsx

```diff
@@ -79,5 +99,10 @@ const getStyles = (theme: GrafanaTheme2) => {
       color: theme.colors.text.secondary,
       marginRight: `${theme.spacing(1)}`,
     }),
+    sizeIndicator: css({
+      margin: '0px',
+      color: querySizeInBytes! > 50000 ? '#FF5286' : '#6CCF8E',
```

Comment:
Hm, I feel the `50000` value is a bit "hidden" here in CSS. What do you think about introducing a const somewhere more prominent?

Also, before merging, I feel we should be sure that 50000 is a "good" number to start with.

Reply:
this makes sense! also disabled the colour change for now until find the best way to calculate the threshold, as discussed in standup.